### PR TITLE
Node roles quickstart.

### PR DESCRIPTION
**Pull request by samscott89**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "oso": ">=0.6.0"
+    "express": "~4.17.1",
+    "oso": ">=0.12.0"
+  },
+  "devDependencies": {
+    "supertest": "^6.1.3"
   }
```
Comment (leina05) [inline package.json:17]:
```suggestion
  },
  "scripts":  {
      "start": "app.js"
  }
```

I needed this for it to run

**Review by leina05 — COMMENTED** [review]
Getting this error when I try to run: 

```
> quickstart@0.0.0 start /Users/leinamcdermott/code/oso-nodejs-quickstart
> app.js

sh: app.js: command not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! quickstart@0.0.0 start: `app.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the quickstart@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/leinamcdermott/.npm/_logs/2021-06-21T14_05_46_916Z-debug.log
```

Someone else should try it--could just be that I suck at Node things.

**Comment by gj** [comment]
@samscott89 can we close this?

**Comment by samscott89** [comment]
> 
> 
> @samscott89 can we close this?

Of course!

**Pull request closed without merging**